app/locales/en/translations.ts

```diff
@@ -529,7 +529,7 @@ export default {
         new_fork_info: 'Your fork is being created. You\'ll receive an email when it is complete.',
         create_fork_modal: 'Are you sure you want to fork this project?',
         unable_to_delete_fork: 'Any child components must be deleted prior to deleting this component.',
-        page_title: '{{nodeTitle}} Forks',
+        page_title: '{{{nodeTitle}}} Forks',
```

Comment:
Just checking, this means we're trusting that titles from the API have already been sanitized? We use our `fixstring` transform to [slightly desanitize](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/utils/fix-special-char.ts#L8-L10) node titles, is there any chance that leaves an opening?

Reply:
Actually, this is because the title was being double-encoded. Once by the translations engine, and again when the translation is used in the template. This just skips the encoding by the translations engine. I will add a comment on these as well.

Reply:
Oh, I thought translations were left as-is when rendered in the template. We can use `<a>` and `<strong>` and such in translation strings, without them being encoded.

Reply:
It might because this is a page title? Anyways: https://github.com/jamesarosen/ember-i18n/issues/77#issuecomment-25253577